guides/v3.7.0/components/defining-a-component.md

```diff
@@ -113,6 +107,17 @@ export default Route.extend({
 {{/each}}
 ```
 
+or 
+
+```handlebars {data-filename=app/templates/index.hbs}
+{{#each this.model as |post|}}
+  {{!-- either foo-component or bar-component --}}
+  {{#let (component (concat this.componentName)) as |Post|}}
```

Comment:
Is this concat left over from something? shouldn't it be:

```suggestion
  {{#let (component this.componentName) as |Post|}}
```

Reply:
done